src/components/WwwFrame/WwwPage.vue

```diff
@@ -103,6 +103,7 @@ export default {
 	height: 100%;
 	display: flex;
 	flex-flow: column nowrap;
+	min-height: 100vh;
```

Comment:
I see that the `height: 100%` rule is still included. Is this working to solve the issue you were seeing?